packages/st2/debian/install

```diff
@@ -6,3 +6,4 @@
 ../st2actions/conf/syslog.*.conf etc/st2/
 ../st2reactor/conf/logging.*.conf etc/st2/
 ../st2reactor/conf/syslog.*.conf etc/st2/
+../conf/nginx usr/share/doc/st2/conf/
```

Comment:
suggest `/st2/conf/nginx`: it's likely we'll get more config examples (e.g. one day we decide support deploying on apache, or integrate with other 3rd party where a config sample needed).
With that, +1.